### DISCUSSION TITLE
Gauges instead of histogram

### DISCUSSION THREAD

**Discussion by aurrelhebert** [kamon-io/Kamon#1124] [Q&A]
Hi all, 

   For some Metrics that are generated by Kamon, I would have prefer a gauges as I am only interested in the current seen distribution (min, max, avg). I was quite happy to find the `gauges` configuration parameter. However, is there a way to delete or filter out the buckets series generated for those metrics? At the moment I have set empty `custom` buckets, and I am still creating at least one `bucket` series. 

Best

**Comment by ivantopo** [comment]
Hey @aurrelhebert, I'm guessing you are talking about the metrics that get exposed in the prometheus reporter?

**Comment by aurrelhebert** [comment]
Hi @ivantopo indead :)

**Comment by aurrelhebert** [comment]
Hello @ivantopo, any news?
